system/jlib/jcomp.cpp

```diff
@@ -814,7 +814,7 @@ void CppCompiler::setTargetBitLength(unsigned bitlength)
     case GccCppCompiler:
         switch (bitlength)
         {
-        case 32: option = "-m32"; break;
```

Comment:
The issue would arise if you installed a 32-bit version of the platform on a 64-bit OS that was configured to support both 64-bit and 32-bit applications.